catalog/ui/search-ui-app/src/main/resources/features.xml

```diff
@@ -17,6 +17,17 @@
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
+    <feature name="catalog-ui" install="auto" version="${project.version}" description="Catalog UI">
+        <feature prerequisite="true">camel-http</feature>
+        <feature prerequisite="true">search-ui-app</feature>
+
+        <configfile finalname="/etc/org.codice.ddf.catalog.ui.config" override="true">
```

Comment:
I think this is tech debt.  These defaults should be set by the catalog UI classes or blueprint and not a config file in the feature file.  This makes it more difficult for other distributions to override these values.  This is probably not the right PR to do that work though.